tests/phpunit/elementor/data/base/mock/template/sub-endpoint.php

```diff
@@ -1,37 +1,18 @@
 <?php
 namespace Elementor\Tests\Phpunit\Elementor\Data\Base\Mock\Template;
 
-class SubEndpoint extends \Elementor\Data\Base\SubEndpoint {
-
-	/**
-	 * @var \Elementor\Tests\Phpunit\Elementor\Data\Base\Mock\Template\Controller
-	 */
-	public $controller;
+use Elementor\Tests\Phpunit\Elementor\Data\Base\Mock\Template\Traits\MockBypassRegister;
```

Comment:
Shorten it

Reply:
Handled: https://github.com/elementor/elementor/pull/12820/commits/324c286c9ee3c5904b543336d66ff4ca235c99e8